packages/ruleset-migrator/src/validation/schema.ts

```diff
@@ -59,8 +59,8 @@ const schema: JSONSchema4 = {
           'json-schema-draft4',
           'json-schema-draft6',
           'json-schema-draft7',
-          'json-schema-2019-09',
-          'json-schema-2020-12',
+          'json-schema-draft-2019-09',
```

Comment:
fwiw this is a "soft" breaking change.
In v5 it's called `json-schema-2019-09`, but it was an oversight that I never really addressed back then because this would be a breaking change.
Now, it's time to make it more consistent.

Reply:
hmm or maybe we keep it as is?
seems like the authors don't use the word draft here https://json-schema.org/draft/2019-09/release-notes.html
@wmhilton thoughts?

Reply:
I think we should keep it as is. @P0lip I don't think its worth the breaking change

Reply:
I can still alias the old value to that new one, so folks can use both

Reply:
> I think we should keep it as is. @P0lip I don't think its worth the breaking change

The primary motivation I have here is that in JS rulesets you'd refer to these formats differently. In JS they're named "jsonSchemaDraft2019_09" and `2020_12` respectively.

AFAIK json-schema-draft-2020-12 doesn't even exist in Spectral v5, so it'd be only 2019-09 that's changed.